sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -207,28 +207,28 @@ def strip_protocol_from_uri(uri):
 
 
 @contextmanager
-def send_trace_context_manager(span_name=SPAN_NAME_SEND):
+def send_trace_context_manager(span_name=SPAN_NAME_SEND, traceparent=None, attributes=None):
```

Comment:
I assume the PR is not completed since the `traceparent` and `attributes` are not used

Reply:
yes - this is more of a draft - we dont need traceparent and attributes here

Reply:
also, this is just to demostrate the usage and verify it's working as expected - https://github.com/Azure/azure-sdk-for-python/pull/16982 this PR would be the one that needs to go in first